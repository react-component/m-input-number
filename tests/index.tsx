import * as React from 'react';
import { render } from 'enzyme';
import renderToJson from 'enzyme-to-json';
import InputNumber from '../src/';

describe('basic', () => {
  it('base.', () => {
    const wrapper = render(
      <InputNumber
        min={-8}
        max={10}
        style={{ width: 100 }}
      />
    );
    expect(renderToJson(wrapper)).toMatchSnapshot();
  });

});
