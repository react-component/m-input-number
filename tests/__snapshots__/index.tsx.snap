// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic base. 1`] = `
<div
  class="rmc-input-number"
  style="width:100px;"
>
  <div
    class="rmc-input-number-handler-wrap"
  >
    <span
      aria-disabled="false"
      aria-label="Increase Value"
      class="rmc-input-number-handler rmc-input-number-handler-up "
      role="button"
      unselectable="true"
    >
      <span
        class="rmc-input-number-handler-up-inner"
        unselectable="true"
      />
    </span>
    <span
      aria-disabled="false"
      aria-label="Decrease Value"
      class="rmc-input-number-handler rmc-input-number-handler-down "
      role="button"
      unselectable="true"
    >
      <span
        class="rmc-input-number-handler-down-inner"
        unselectable="true"
      />
    </span>
  </div>
  <div
    aria-valuemax="10"
    aria-valuemin="-8"
    class="rmc-input-number-input-wrap"
    role="spinbutton"
  >
    <input
      autocomplete="off"
      class="rmc-input-number-input"
      max="10"
      min="-8"
      step="1"
      value=""
    />
  </div>
</div>
`;
